File: 프로그래머스/unrated/161990. 바탕화면 정리/바탕화면 정리.java
import java.util.*;

class Solution {
    public int[] solution(String[] wallpaper) {
       List<Integer> lowList = new ArrayList<>();
		List<Integer> columnList = new ArrayList<>();
		
		for(int i = 0; i < wallpaper.length; i++) {
			if(wallpaper[i].contains("#")) {
				lowList.add(i);
				columnList.add(wallpaper[i].lastIndexOf("#")); 				
				columnList.add(wallpaper[i].indexOf("#")); 				
			}
			
		}
		
		return new int[] {Collections.min(lowList), 
				Collections.min(columnList), 
				Collections.max(lowList) + 1, 
				Collections.max(columnList) + 1};
    }
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    let rows = [];
    let columns = [];
    for (let i = 0; i < wallpaper.length; i++){
        if (wallpaper[i].includes("#")){
            rows.push(i);
            columns.push(wallpaper[i].indexOf('#'));
            columns.push(wallpaper[i].lastIndexOf('#'));
        }
    }
    return [Math.min(...rows), Math.min(...columns), Math.max(...rows) + 1, Math.max(...columns) + 1];
}